doozer/doozerlib/backend/konflux_client.py

```diff
@@ -433,9 +433,9 @@ def _modify_param(params: List, name: str, value: Union[str, bool, list[str]]):
             }]
 
         if not sast:
-            params = [p for p in params if p.get("name") not in ("sast-unicode-check", "sast-shell-check")]
-
-        obj["spec"]["params"] = params
+            self._logger.info("Removing sast-unicode-check and sast-shell-check tasks since sast is disabled")
+            tasks = [task for task in obj["spec"]["pipelineSpec"]["tasks"] if task.get("name") not in ("sast-unicode-check", "sast-shell-check")]
```

Comment:
it would be good to know if something got filtered
we can compare the length of before and after and then log appropriately